ci/azure/conda_linux.yml

```diff
@@ -0,0 +1,57 @@
+parameters:
+  name: ''
+  vmImage: ''
+
+jobs:
+- job: ${{ parameters.name }}
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+  strategy:
+    matrix:
+      Python35:
+        python.version: '35'
+      Python36:
+        python.version: '36'
+        coverage: true
+      Python37:
+        python.version: '37'
+      Python38:
+        python.version: '38'
+
+  steps:
+  - bash: echo "##vso[task.prependpath]/usr/share/miniconda/bin"
+    displayName: Add conda to PATH
+  - script: conda env create --quiet --file ci/requirements-py$(python.version).yml
+    displayName: Create Anaconda environment
+  - script: |
+      source activate test_env
+      pip install pytest-azurepipelines
+      pip install -e .
+    displayName: 'pip dependencies'
+  - script: |
+      source activate test_env
+      conda list
+    displayName: 'List installed dependencies'
+  - script: |
+      source activate test_env
+      export NREL_API_KEY=$(nrelApiKey)
+      pytest pvlib --remote-data --junitxml=junit/test-results.xml --cov --cov-report=xml --cov-report=html
+    displayName: 'pytest'
+  # - script: |
+  #     source activate test_env
+  #     flake8 pvlib
+  #   displayName: 'flake8'
```

Comment:
delete?

Reply:
Done. I was wondering if there was a reason it was left there.